bin/raptor

```diff
@@ -94,17 +95,22 @@ parent
     transform: utils.toInt,
     callback: utils.validate('connectionTimeout', utils.validator.isInt)
   })
+  .option('logcat', {
```

Comment:
I think thie logcat option should be available at the raptor level not the --test level, because it's not test specific, and also --metrics is at the top level help